bin/vyper

```diff
@@ -53,7 +54,7 @@ if __name__ == '__main__':
         for f in uniq(args.format.split(',')):
             formats.append(translate_map.get(f, f))
 
-        out_list = compiler.compile(codes, formats, output_type='list')
+        out_list = vyper.compile(codes, formats, output_type='list')
```

Comment:
Can `output_type` logic be handled in this script instead?. e.g. return it as a dict and handle list conversion here.

Reply:
As mentioned above, I prefer having this as part of the vyper module, otherwise nobody else can do multiple input / multiple output (vyper-debug for example). 
As it stands compile does support a single string input.